src/styles/theme.css

```
@@ -1,12 +1,79 @@
+  /* v2 dashboard page background — light default, overridden for .dark below */
+  --color-surface-main: #eeeff3;
+
+  /* v2 telegram-ui card surface — light default, overridden for .dark below */
+  --color-surface-primary: #ffffff;
+
+  /* Success green — opacity applied at the call site via color-mix() */
+  --color-success: #25a761;
+
+  --color-surface-glass: #FFFFFF99
+
+  --color-surface-tertiary: #E2E8F0
```

Stylelint (17.14.0)

[error] Missing var function for "--color-surface-tertiary" (custom-property-no-missing-var-function)

(custom-property-no-missing-var-function)